tests/test_app/parser.h

```diff
@@ -73,6 +73,8 @@ struct IOInfo {
 };
 
 struct KernelInfo {
+    int temp[100];
```

Comment:
As a said, this should not be merged in master branch. I can help you debug the buffer overflow problem.